app/src/main/java/org/mozilla/fenix/home/intent/DeepLinkIntentProcessor.kt

```diff
@@ -89,6 +92,16 @@ class DeepLinkIntentProcessor(
                 if (SDK_INT >= Build.VERSION_CODES.N) {
                     val settingsIntent = Intent(Settings.ACTION_MANAGE_DEFAULT_APPS_SETTINGS)
                     activity.startActivity(settingsIntent)
+                } else {
+                    activity.openToBrowserAndLoad(
+                        searchTermOrURL = SupportUtils.getSumoURLForTopic(
+                            activity,
+                            SupportUtils.SumoTopic.SET_AS_DEFAULT_BROWSER
+                        ),
+                        newTab = true,
+                        from = BrowserDirection.FromGlobal,
+                        flags = EngineSession.LoadUrlFlags.external()
```

Comment:
Because this is an external deep link. Nice!

Reply:
Thanks!